config/formats.js

```diff
@@ -541,7 +541,21 @@ let Formats = [
 		},
 		onModifyPriority: function (priority, pokemon, target, move) {
 			// @ts-ignore
-			if (move.category !== 'Status' && pokemon && pokemon.forte) return priority + pokemon.forte.priority;
+			if (move.category !== 'Status' && pokemon && pokemon.forte) {
+				let abilPriority = pokemon.getAbility().onModifyPriority;
+				let movePriority = priority;
+				// @ts-ignore
+				let fortePriority = pokemon.forte.priority;
+				if (abilPriority) {
+					let abilMoveCall = abilPriority.call(this, priority, pokemon, target, move);
+					// @ts-ignore
+					let abilForteCall = abilPriority.call(this, priority, pokemon, target, pokemon.forte);
+					// @ts-ignore
+					if (typeof abilForteCall === 'number') fortePriority = abilForteCall;
+					if (typeof abilMoveCall === 'number') movePriority = abilMoveCall;
```

Comment:
You're applying the ability priority twice?

(Three times, when including the actual ability activation?)

That seems wrong...

Reply:
the ability activation doesnt happen because the format's `onModifyPriority` function overrides it, that's why it's being checked. I'll just change it so this isnt controversial

Reply:
How does a format event handler override an ability event handler for a relay event?

Especially since the format event handler is additive?

Did you mean to say that a non-additive ability event handler overrides the format event handler? If so, changing the format event handler wouldn't fix it anyway.